animeal/src/Flows/Main/Modules/Home/Main/HomeViewModel.swift

```diff
@@ -194,7 +197,17 @@ final class HomeViewModel: HomeViewModelLifeCycle, HomeViewInteraction, HomeView
                 self.onFeedingPointsHaveBeenPrepared?(viewItems)
                 self.feedingStatus = result.feedingStatus
                 self.onFeedingHaveBeenCompleted?()
-                self.coordinator.routeTo(.feedingComplete)
+
+                let userAttributes = try await userProfileService.fetchUserAttributes()
```

Comment:
Hi @mgazizova , Can you check if the pipeline build failure message is a false alarm or is it legit. Error from logs attached.
CC: @rinold 
 /Users/runner/work/animeal_iOS/animeal_iOS/animeal/src/Flows/Main/Modules/Home/Main/HomeViewModel.swift:201:48: error: reference to property 'userProfileService' in closure requires explicit use of 'self' to make capture semantics explicit
2023-08-25T15:23:26.0179460Z                 let userAttributes = try await userProfileService.fetchUserAttributes()